spec/swagger.yaml

```diff
@@ -7756,6 +7896,89 @@ definitions:
         description: Should send email notification to this email.
         example: true
 
+  WebPageListRequest:
+    type: object
+    properties:
+      urlPath:
+        type: string
+        description: url
+        example: /home/page.htm
+      urlPathExactMatch:
+        type: boolean
+        description: Should apply exact match or not.
+        example: false
+      domainName:
+        type: string
+        description: domain name
+        example: my.domain.com
+      urlCaptureDateAfter:
+        type: string
+        format: date-time
+        example: '2020-11-21T01:51:17Z'
+        description: The date after which the url was captured.
```

Comment:
The date after which the URL was captured.

Reply:
Done